.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -38,3 +38,7 @@ Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
 [user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 output here
 ```
+
+### Priorities
```

Comment:
@sternenseemann one could just remove the word "priorities" to avoid unwanted associations.

This is what I thought initially anyway, it should just be a note, a nudge to action.

```suggestion
---
```

Reply:
Yeah, still think priorities is misleading.

Reply:
Also this should be in a comment and not show up in the actual posted issue (like with the PR template).

Reply:
Oh but the point is exactly for readers to see it. Otherwise only issue authors would be informed of the mechanism.